### PR TITLE
Microsoft Exchange Message Trace: fix httpjson cursor template

### LINKED ISSUE #5180: Microsoft Exchange Message Trace httpjson template broken cursor usage
The httpjson input template irrationally/unnecessarily attempts to use parseDuration with no arguments, which filebeat does not accept.

e.g. template snippet,

```
cursor:
  last_execution_datetime:
    value: '[[(formatDate (now (parseDuration)) "RFC3339")]]'
```

Error,

Screenshot 2023-02-04 at 6 07 50 pm

This means that the request.transforms reference to the cursor will fail, and the default value which applies the parseDuration minus {{initial_interval}} gets used.

Hence the integration continually requests historic logs and never really updates the cursor to only request logs from the last time it ran. e.g.

```
request.transforms:
  - set:
      target: url.params.$filter
      value: StartDate eq datetime'[[.cursor.last_execution_datetime]]' and EndDate eq datetime'[[(formatDate (now) "RFC3339")]]'
      default: StartDate eq datetime'[[(formatDate (now (parseDuration "-{{initial_interval}}")) "RFC3339")]]' and EndDate eq datetime'[[(formatDate (now) "RFC3339")]]'
```

PR inbound shortly.

### PR DESCRIPTION
**Pull request by colin-stubbs**
Bug Fix

## What does this PR do?

Fix incorrect use of parseDuration with zero arguments when setting cursor value for httpjson input.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

Make not broke.

## How to test this PR locally

Test against a Microsoft Exchange Online tenant.

## Related issues

- Closes #5180

## Screenshots

Not applicable.

### CODE REVIEW

**Comment by efd6** [comment]
Please run `elastic-package build`.

File: packages/microsoft_exchange_online_message_trace/data_stream/log/agent/stream/httpjson.yml.hbs [hunk 1]
```diff
@@ -56,9 +56,9 @@ response.pagination:
       value: '[[ (add (toInt (index .last_response.url.params "$skiptoken")) 2000) ]]'
 cursor:
   last_execution_datetime:
-    value: '[[(formatDate (now (parseDuration)) "RFC3339")]]'
+    value: '[[(formatDate (now) "RFC3339")]]'
```
Comment (kcreddy) [inline packages/microsoft_exchange_online_message_trace/data_stream/log/agent/stream/httpjson.yml.hbs:59]: With this change, the `value` which is being used by `StartDate` seems to be same as `EndDate`. Instead, you will need to extract the timestamp from the response using `first_event` or `last_event` depending on the sort order.

File: packages/microsoft_exchange_online_message_trace/changelog.yml [hunk 2]
```diff
@@ -1,3 +1,8 @@
+- version: "0.2.1"
+  changes:
+    - description: Fix issue #5180
```
Comment (kcreddy) [inline packages/microsoft_exchange_online_message_trace/changelog.yml:3]: Briefly describe the issue

**Pull request closed without merging**